README.md

```diff
@@ -32,21 +573,44 @@ tests for both .NET desktop and .NET Core.
 On both Windows and all other supported operating systems, you can run
 `dotnet build` to build and `dotnet test` to run the tests.
 
+
 ## Origin
 
 This project began life as a [Sharpen][sharpen]-based auto-port from
 [jsonld-java][jsonld-java].
 
-  [jsonld]:         https://json-ld.org/
-  [sharpen]:        http://community.versant.com/Projects/html/projectspaces/db4o_product_design/sharpen.html
-  [jsonld-java]:    https://github.com/jsonld-java/jsonld-java
-  [nuget]:          https://www.nuget.org/packages/json-ld.net/
-  [nuget-badge]:    https://img.shields.io/nuget/v/json-ld.net.svg
-  [coc]:            https://opensource.microsoft.com/codeofconduct/
-  [coc-faq]:        https://opensource.microsoft.com/codeofconduct/faq/
-  [ms-mail]:        mailto:opencode@microsoft.com
-  [dnc]:            https://dot.net
-  [dnc-tutorial]:   https://www.microsoft.com/net/core
-  [codecov]:        https://codecov.io/gh/linked-data-dotnet/json-ld.net
-  [codecov-badge]:  https://img.shields.io/codecov/c/github/linked-data-dotnet/json-ld.net/master.svg
-  [gha-badge]:      https://github.com/linked-data-dotnet/json-ld.net/workflows/dotnet/badge.svg
+Documentation for this library is in part drawn from https://github.com/linked-data-dotnet/json-ld.net
+
+  [coc]:                        https://opensource.microsoft.com/codeofconduct/
+  [coc-faq]:                    https://opensource.microsoft.com/codeofconduct/faq/
+  [codecov]:                    https://codecov.io/gh/linked-data-dotnet/json-ld.net
+  [codecov-badge]:              https://img.shields.io/codecov/c/github/linked-data-dotnet/json-ld.net/master.svg
+
+  [ms-mail]:                    mailto:opencode@microsoft.com
+  [dnc]:                        https://dot.net
+  [dnc-tutorial]:               https://www.microsoft.com/net/core
+
+  [gha-badge]:                  https://github.com/linked-data-dotnet/json-ld.net/workflows/dotnet/badge.svg
+
+  [jsonld]:                     https://json-ld.org/
+  [jsonld-java]:                https://github.com/jsonld-java/jsonld-java
+
+  [JSON-LD 1.0]:                http://www.w3.org/TR/2014/REC-json-ld-20140116/
```

Comment:
Can we try to keep the link references in [kebab case](https://en.wiktionary.org/wiki/kebab_case), please? :)

```suggestion
  [json-ld-1.0]:                http://www.w3.org/TR/2014/REC-json-ld-20140116/
```

Reply:
Yeah I did wonder about this (as with some of your suggestions above). At the end of the day I tried to stick as closely as possible to the content f json-ld.js so that we can easily update our content from there e.g. when new versions of json-ld emerge and he updates their documentation. I'll update the casing of the link references though becuase it was bugging me too...